pkg/validation/validation.go

```diff
@@ -165,8 +166,58 @@ func validateDistro(distro string, dev, released kdm.Data) error {
 				logrus.Infof("the release: %v", release)
 				return fmt.Errorf("failed to validate RKE2 charts: %v", err)
 			}
+			if err := validateEncryptedKeyRotation(release); err != nil {
+				return fmt.Errorf("failed to validate rke2 encrypted key rotation: %v", err)
+			}
+		}
+	}
+
+	if distro == utiliies.K3S {
+		raw, _, err := unstructured.NestedSlice(dev.K3S, "releases")
+		if err != nil {
+			return err
 		}
+		for _, r := range raw {
+			release, ok := r.(map[string]interface{})
+			if !ok {
+				return err
```

Comment:
What error is being returned here?

Reply:
It was supposed to return a new error, fixed